packages/hash/frontend/src/pages/[account-slug]/shared/white-card.tsx

```diff
@@ -43,19 +65,14 @@ export const WhiteCard = ({
           : {},
       ]}
     >
-      {onClick || href ? (
-        <CardActionArea
-          {...(onClick ? { onClick } : { href })}
-          disableRipple
-          disableTouchRipple
-          sx={{
-            [`&:hover .${cardActionAreaClasses.focusHighlight}`]: {
-              opacity: 0,
-            },
-          }}
-        >
+      {onClick ? (
+        <WhiteCardActionArea onClick={onClick}>
           {cardContent}
-        </CardActionArea>
+        </WhiteCardActionArea>
+      ) : href ? (
+        <Link href={href} sx={{ textDecoration: "none" }}>
+          <WhiteCardActionArea>{cardContent}</WhiteCardActionArea>
+        </Link>
```

Comment:
Have you considering using the `component` prop to `CardActionArea` to change the component from `"a"` to a `Link` when passing href? Seems a simpler solution. 

`CardActionArea` is based on `ButtonBase`, and this is how we make Buttons use `Link` elsewhere.

Reply:
Great suggestion, I didn't think of that. Have addressed in https://github.com/hashintel/hash/pull/1564/commits/bfd0ca1118fc5d97364de6e0c42c4db9d483a3e8